features/step_definitions/journey/registration_steps.rb

```diff
@@ -97,9 +97,16 @@
   expect(page).to have_content("Registration complete")
 
   @reg_number = @journey.confirmation_page.registration_number.text
-  puts "Registration #{@reg_number} created successfully"
+  find_text = [@reg_number]
```

Comment:
Nice 😍 One suggestion, what do you think?

```suggestion
  find_text = [@reg_number, @business_name]
```

Reply:
That will break tests as we don't have a business name in the email :)

Reply:
Ah OK, good point 😁